src/hooks/__tests__/useRadio.test.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FC } from 'react'
+import { render, wait } from '@vtex/test-tools/react'
+
+import { useRadio } from '../useRadio'
+import { FormContext } from '../../components'
+
+const toFixed = (value: number, precision: number): string => {
+  const power = Math.pow(10, precision || 0)
+  return String(Math.round(value * power) / power)
+}
```

Comment:
This seems to be used on multiple files. Can you extract this to an auxiliary file?